### PR TITLE
Setup intake enum field values

### PR DESCRIPTION
**Pull request by hamza-yusuff**
## Notion ticket link

[Setup intake enum field values](https://www.notion.so/uwblueprintexecs/Setup-intake-enum-field-values-5b93660c622a4efc94ec25d79059d410)

## Implementation description
* Updated enum values in `intake` and `caregiver`
* Added new enum values in `intake`

## Steps to test
1.

## What should reviewers focus on?
* 

## Checklist
- [X] My PR name is descriptive and in imperative tense
- [X] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [X] I have run the appropriate linter(s)
- [X] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/python/app/models/caregiver.py [hunk 1]
```diff
@@ -8,6 +8,9 @@
     "FOSTER_MOM",
     "FOSTER_DAD",
     "GRANDPARENT",
+    "FOSTER CAREGIVER",
+    "KINSHIP CAREGIVER",
+    "BIOLOGICAL FAMILY",
```
Comment (jennchenn) [inline backend/python/app/models/caregiver.py:13]: these should be the only options in this enum

can you also update to have `_` between the words?

File: backend/python/app/models/intake.py [hunk 2]
```diff
@@ -3,7 +3,27 @@
 from . import db
 from .base_mixin import BaseMixin
 
-court_status_enum = db.Enum("APPROVE", name="intakes_court_status")
+court_status_enum = db.Enum(
+    "APPROVE",
```
Comment (jennchenn) [inline backend/python/app/models/intake.py:7]: remove

File: backend/python/app/models/intake.py [hunk 3]
```diff
@@ -3,7 +3,27 @@
...
+    "INTERIM CARE",
+    "FINAL ORDER FOR SOCIETY CARE",
+    "EXTENDED SOCIETY CARE",
+    "SUPERVISION ORDER",
+    "KIN SERVICE PLACEMENT",
+    "LIVING WITH BIO FAMILY",
+    "OTHER",
+    name="intakes_court_status",
```
Comment (jennchenn) [inline backend/python/app/models/intake.py:15]: please update to have `_` between words

File: backend/python/app/models/intake.py [hunk 4]
```diff
@@ -3,7 +3,27 @@
...
+)
+
+first_nation_heritage_enum = db.Enum(
+    "FIRST NATION REGISTERED",
```
Comment (jennchenn) [inline backend/python/app/models/intake.py:19]: add `_` between words

**Review by jennchenn — CHANGES_REQUESTED** [review]
looking good! few updates + please change all enum values to be snake cased

**Comment by hamza-yusuff** [comment]
> looking good! few updates + please change all enum values to be snake cased

So if it's all caps and has '_' instead of space, it would still be considered snake cased right?

File: backend/python/app/models/intake.py (hunk 2 above)
Comment (hamza-yusuff) [reply backend/python/app/models/intake.py]: it's only "APPROVE" that need to be removed right?

**Review by jennchenn — APPROVED** [review]
lgtm!

**Pull request merged**